### PR TITLE
Feature/browse files

### PR DESCRIPTION
**Pull request by MpiloMshengu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "0.19.0",
     "node-sass": "4.12.0",
     "react": "16.9.0",
     "react-dom": "16.9.0",
```
Comment (Andileh) [inline package.json]: remove this big space

File: src/components/functions.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+export default  function optimizeSvg(dataUrl){
+  return axios.post(`http://localhost:3500/optimizeSvg`, { dataUrl })
```
Comment (sjumba) [inline src/components/functions.js]: cant you use a variable for http://localhost:3500/ , and define it in one place and use it throughout.

File: src/components/chooseFolder/chooseFolder.js [hunk 3]
```diff
@@ -1,24 +1,30 @@
...
 
-  onChangeHandler = event => {
-    this.setState({ path: event.target.files });
-  };
-
+  onChangeHandler=event=>{
+     this.setState({path:event.target.files});
+  }
+  
+  async optimize(){
```
Comment (CastelloDev) [inline src/components/chooseFolder/chooseFolder.js]: es6 syntax please

File: src/components/chooseFolder/chooseFolder.js [hunk 4]
```diff
@@ -1,24 +1,30 @@
...
+   var data= await optimizeSvg(this.state.dataUrl);
```
Comment (CastelloDev) [inline src/components/chooseFolder/chooseFolder.js]: please use let

File: src/components/chooseFolder/chooseFolder.js [hunk 5]
```diff
@@ -1,24 +1,30 @@
...
+    }
+    this.optimize = this.optimize.bind(this);
```
Comment (CastelloDev) [inline src/components/chooseFolder/chooseFolder.js]: this is old react syntax

File: src/constants/constants.js [hunk 6]
```diff
@@ -1,3 +1,3 @@
-export const IMPORT_DIRECTORY = './src/svgs/';
-export const EXPORT_DIRECTORY = './src/optimizedSVGs/';
-export const CODE_ELEMENT = '<defs><style>.colour-class{fill:#FFFFFF;}</style></defs>';
\ No newline at end of file
+const BASE_URL = 'http://localhost:3500/';
+
```
Comment (CastelloDev) [inline src/constants/constants.js]: please add endpoint name here as well as a separate constant

**Pull request merged**